docs/production/ingesters-with-wal.md

```diff
@@ -29,6 +29,34 @@ _The WAL is currently considered experimental._
 
 2. As there are no transfers between ingesters, the tokens are stored and recovered from disk between rollout/restarts. This is [not a new thing](https://github.com/cortexproject/cortex/pull/1750) but it is effective when using statefulsets.
 
+## Disk space requirements
+
+Based on tests in real world:
+
+* Numbers from an ingester with 1.2M series, ~80k samples/s ingested and ~15s scrape interval.
+* Checkpoint period was 20mins, so we need to scale up the number of WAL files to account for the default of 30mins.
+* At any given point, we have 2 complete checkpoints present on the disk and a 2 sets of WAL files between checkpoints (and now).
+* This peaks at 3 checkpoints and 3 lots of WAL momentarily, as we remove the old checkpoints.
+
+|  |  |
+|:-|:-|
+| Size of 1 checkpoint for 1.2M series | 1410 MiB |
+| Avg checkpoint size per series | 1.2 KiB |
+| No. of WAL files between checkpoints (30m checkpoint) | 30 mins x 87 / 20mins = 130 |
+| Size per WAL file | 32 MiB (reduced from Prometheus) |
+| Total size of WAL | 4160 MiB |
+| Steady state usage | 2 x 1410 + 2 x 4160 = ~11 GiB |
+| Peak usage | 3 x 1410 + 3  x 4160 = ~17 GiB |
```

Comment:
Units missing here. Math seems wrong: 3 * 1410 MiB + 3  * 4160 MiB = 16.32 GiB

Reply:
Rounding off to the next number seems safer when planning the resources, Maybe we could have 16.5, but would prefer 17. I will add the units there.

Reply:
"Rounding" from 16.32 GiB to 17 GiB is wrong. We can write ~16.3 GiB, which would be correct, and leave rounding up for planning the resources to the reader, who will surely be smart enough to add extra capacity.